projects/storefrontlib/src/lib/store-finder/components/store-finder-store-description/store-finder-store-description.component.ts

```diff
@@ -20,7 +17,6 @@ export class StoreFinderStoreDescriptionComponent
   isLoading$: Observable<any>;
```

Comment:
Please check `any` in all changed files to see whether we can give a explicitly type.

Reply:
In this case i added `boolean`. Btw i think that i have seperate task for it #415